loom/common/assets/assets.h

```diff
@@ -101,6 +101,8 @@ enum LoomAssetType
     LATMesh         = LOOM_FOURCC('M', 'S', 'H', 1),
     LATScript       = LOOM_FOURCC('L', 'O', 'O', 'M'),
     LATBinary       = LOOM_FOURCC('B', 'I', 'N', 1),
+    LATVertexShader = LOOM_FOURCC('V', 'E', 'R', 'T'),
```

Comment:
If the deserialization logic needed something special then you want the type... otherwise if you just need to load some UTF8 bytes LATText you're probably good.